### DISCUSSION TITLE
是否可以支持ipad

### DISCUSSION THREAD

**Discussion by iyexiaolou** [Semporia/TikTok-Unlock#308] [Q&A] [answered]
目前这个方法无法在ipad上使用

**Comment by Semporia** [comment, marked as answer]
> 目前这个方法无法在ipad上使用

iPad Wi-Fi版還要解鎖？

不應該直接就食用嘛？

**Reply by 9-yun** [reply]
不能食用      麻烦大佬想想办法

**Reply by Semporia** [reply]
可以直接食用，添加分流即可！

**Reply by 9-yun** [reply]
> 可以直接食用，添加分流即可！


大佬辛苦！俺是一个纯小白，所以问的问题可能很傻逼，希望大佬还有耐心，俺还有几个问题，问题1:对于quantumult x 的方法一（即步骤3），步骤4和步骤5添加的东西大佬能帮忙看下是那样添加吗？    问题2:我刚自己买了一个quantumult x ，但是步骤6的“找到TikTok策略”这个东西我没找到，在哪呀？  麻烦大佬
![image](https://user-images.githubusercontent.com/114273864/220888694-e17d96a9-01e5-4ac6-adfa-71a1395cbed2.jpeg)

![image](https://user-images.githubusercontent.com/114273864/220888776-fa2d778c-c27f-4627-aa86-0f8378db67a1.jpeg)

![image](https://user-images.githubusercontent.com/114273864/220888832-d7d1f04d-700b-4d32-bf88-66537a93dc9f.jpeg)

![image](https://user-images.githubusercontent.com/114273864/220888880-b580eac6-854a-4d8d-af86-bbfdaa3bc500.jpeg)

**Comment by Mohamme-ui** [comment]
> أفضل جهاز iPad لجهاز iPad

POP
